zerver/openapi/zulip.yaml

```diff
@@ -9066,6 +9075,16 @@ paths:
 
                     **Changes**: New in Zulip 5.0 (feature level 86).
                   example: unicode_emoji
+                scheduled_end_time:
+                  type: string
```

Comment:
timestamps are passed with the type `integer`. see other timestamps in this file.